### DISCUSSION TITLE
How to migrate from V1 to V2 on vanilla JS

### DISCUSSION THREAD

**Discussion by Alim32** [WalletConnect/walletconnect-monorepo#2862] [Developer Support] [answered]
Hey,

I've been looking around for quite a bit now and can't really seem to find a way to upgrade from v1 to v2 on vanilla js/html dapp (nor my C# dapp).

I believe I formerly used a CDN with the web3-provider.

**Comment by boidushya** [comment, marked as answer]
Hey, we have [stopped supporting](https://docs.walletconnect.com/2.0/advanced/migration-from-v1.x/dapps/#web3-provider) `@walletconnect/web3-provider` in favour of `@walletconnect/ethereum-provider`

If you want to specifically use a CDN, we recommend switching to [web3modal](https://docs.walletconnect.com/2.0/web/web3modal/html/wagmi/installation#add-packages-cdn)

Closing this discussion for now, feel free to open if you have further questions

**Reply by Alim32** [reply]
Hmm for some reason I couldn't open that link before it always sent me to the react part of the guide. Im gonna check it out thanks!

**Reply by boidushya** [reply]
Thanks for reporting that, will get it fixed on docs side!

**Reply by Alim32** [reply]
It was the "HTML" button on https://docs.walletconnect.com/2.0/web/about-web3modal under getting started that linked me to React. Anyway I managed to get it all working now thanks.
